src/constants.ts

```diff
@@ -73,7 +73,7 @@ export const defaultStoredProcedure =
     if (!isAccepted) throw new Error('The query was not accepted by the server.');
 };` ;
 
-export const emptyPartitionKeyValue = {};
+export const emptyPartitionKeyValue: string = '';
```

Comment:
Seemed excessive to have a constant for this in the first place, but seems even more excessive if it's just an empty string